static-analysis/action.yml

```diff
@@ -30,6 +30,26 @@ inputs:
     description: 'Version of node to install on the host'
     required: false
     default: 'lts/*'
+  skip_lint_modules:
+    description: skip linter for modules when set to true
+    required: false
```

Comment:
required is not needed when steps are providing a default, the same apply to the other ones below